### PR TITLE
feat: add image paste functionality to text area

### PR DESCRIPTION
**Pull request by jordan-ae**
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by jordan-ae** [comment]
@humansinstitute everything is working to spec but the endpoint for uploads keeps returning a 404. Not sure what's up implementation is on point. Can we double check the endpoint please? I've been debugging for hours now and atp I'm not sure what's going on

[Screencast from 2024-12-20 14-43-22.webm](https://github.com/user-attachments/assets/a826a908-dbd0-478c-8d10-274af1cf52c1)

**Comment by humansinstitute** [comment]
@tobi-bams can you review and advise on the endpoint?

File: src/components/common/TicketEditor/TicketEditor.tsx [hunk 1]
```diff
@@ -279,15 +286,30 @@ const TicketEditor = observer(
                 >
                   Copy
                 </ActionButton>
+                <ActionButton
+                  color="primary"
+                  onClick={handlePreview}
+                  disabled={isCopying}
+                  data-testid="copy-description-btn"
+                >
+                  {isPreview ? 'Preview' : 'Edit'}
```
Comment (tobi-bams) [inline src/components/common/TicketEditor/TicketEditor.tsx:295]: If it is in preview state, the Edit button should be what is visible and when it's in Edit phase the button should say Edit.

**Review by tobi-bams — CHANGES_REQUESTED** [review]

**Comment by tobi-bams** [comment]
> @humansinstitute everything is working to spec but the endpoint for uploads keeps returning a 404. Not sure what's up implementation is on point. Can we double check the endpoint please? I've been debugging for hours now and atp I'm not sure what's going on
> 
>  Screencast.from.2024-12-20.14-43-22.webm

@jordan-ae you are correct, So sorry the mistake is from my end.

The `/public_pic` endpoint has been removed, we now use the `meme_upload` endpoint. But don't worry I will handle this myself. Currently the `meme_upload` is not working properly because we've fully migrated to v2. I need to chat Evan up.
@humansinstitute

**Comment by humansinstitute** [comment]
@tobi-bams - thanks for letting me know. Can you tag me when thats done and we can merge this? 

looking forward to getting this functionality!

**Comment by humansinstitute** [comment]
@tobi-bams - I'm going to merge this and payout bounty for now. 

I think we can revisit and update the endpoint when you have had a chance to talk to Evan.

**Pull request merged**